docs/skopeo.1.md

```diff
@@ -22,7 +34,7 @@ Most commands refer to container images, using a _transport_`:`_details_ format.
   An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$HOME/.docker/config.json`, which is set e.g. using `(docker login)`.
```

Comment:
This part about the default auth location is changing with @umohnani8's submit or do you want to touch it up now?  If not changing, at least for the moment, I'd remove the 'e.g.' here.

Reply:
I think we should add `kpod login" AFTER we have completed the rename and introduce it.

Reply:
That's probably best.